source/server/getting-started/configuration/sponge-conf.rst

```diff
@@ -13,464 +13,1165 @@ Below is a list of available config settings and their comments inside the globa
 will not be filled immediately, and may optionally be added to the file when the server encounters them. There's also
 full example of an unmodified ``global.conf`` file at the bottom of this page, below the following list:
 
-Global Properties of Sponge
-~~~~~~~~~~~~~~~~~~~~~~~~~~~
+GlobalConfig
+============
 
-block-capturing
-===============
+**broken-mods**
```

Comment:
This new format seems to generate a list of the categories, which precedes the actual detailed descriptions. I think this bit is redundant and repetitious.

Reply:
The `Category` suffix is part of the actual class name.